### DISCUSSION TITLE
JLReq-d 編集会議 2024-12-11

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq-d#74] [General]
# JLReq-d 編集会議 2024-12-11

### 出席者
敏先生 @KobayashiToshi, 山口さん @yamahige, 山本さん @taroyamamoto-451, 木田 @kidayasuo 

### ノート
- 箇条書きをドラフトした（下）。[チェックイン済み](https://github.com/w3c/jlreq-d/blob/gh-pages/drafts/4.%20Basic%20Architecture.md#9x-箇条書き)。
  - 箇条書きについて著者向けに書くことはないので、実装者向け。「9章 段落処理」の中。
- 次回1/22水曜日9時から。山本さんも参加予定なので、「2.4フォント：欧文書体の合わせ方」をやる

----

### 9.x 箇条書き

箇条書きはラベルと本体で構成される。ラベルにはビュレットのような同一の記号を繰り返し用いる場合と、順序を表す文字を用いる場合とがある。箇条書きは階層的に用いられる場合もある。

同一の記号はビュレット、中点、米印、アステリスク、ダッシュ類、矢印、などが使われる。

順序を表す文字には次のような例がある。
	•	アラビア数字（1, 2, 3 など）
	•	ローマ字（A, B, C など）
	•	ローマ数字（I, II, III など）
	•	漢数字（一、二、三 など）
	•	日本語のひらがな、カタカナ（あ、い、う や ア、イ、ウ など）
	•	丸中数字①などの囲みつき文字
また、これらの文字を括弧類などで囲む場合や、末尾にピリオドなどの文字を加える場合がある。また、階層構造を示すために、複数の順序を表す文字をピリオドなどの文字で区切る場合がある。

ラベルに順序を表す文字を用いる場合のラベルの配置方法には次の二通りがある
1. ラベルの末尾位置、および本体の開始位置を決める
2. ラベルの先頭位置を決める。この場合にはラベルの末尾と本体との間隔を設定する
どちらの場合でも本体の二行目以降のインデントを設定できる必要がある。

順序を表す文字を括弧類で囲む場合、括弧類に含まれる空白を取り除くか保存するかの選択ができる必要がある。本体の先頭および後続行の先頭は行頭として扱い、これらは5章で述べた括弧類の処理に従う。

#### 縦組みの箇条書き
縦組におけるラベルには横書きにない処理が必要となる。

アラビア数字、ローマ字、ローマ数字は縦中横にし、全体を中央揃えにする。これらの文字を括弧類などで囲む場合、アラビア数字、ローマ字、ローマ数字だけを縦中横にする方法と、括弧類を含めて縦中横にする方法とがある（図示）。

順序を表す文字の末尾にピリオドなどの文字を用いる場合、ピリオドなどの文字は縦中横にしない（図示）。この場合一般的にはピリオドではなく、読点が使われる。

**Comment by kidayasuo** [comment]
> 1. ラベルの末尾位置、および本体の開始位置を決める
> 2. ラベルの先頭位置を決める。この場合にはラベルの末尾と本体との間隔を設定する

のところですが、もう少し詳しく、下のような内容の説明を加えるのはどうでしょう？
- 1の場合にはラベルは右詰めにされること。2の場合にはラベルは左詰めにされること
- 左詰めが欧文にないなどの理由で現時点で実装されていないなら、左詰めの必要性を感じてもらう必要がないでしょうか。例えば、ラベルに漢数字を使う場合には長さの変動が大きいので2の有用、とか、もしくは他の理由を加えられないでしょうか。

>順序を表す文字を括弧類で囲む場合、括弧類に含まれる空白を取り除くか保存するかの選択ができる必要がある。本体の先頭および後続行の先頭は行頭として扱い、これらは5章で述べた括弧類の処理に従う。

これまで「場合がある」という表現で来ていて、突然「選択できる必要がある」は少し違和感がありました。

> アラビア数字、ローマ字、ローマ数字は縦中横

「アラビア数字、ローマ字、ローマ数字など、縦書きで横倒しにされる文字は縦中横にして正立させる」と書き加えるのはどうでしょう。それによってなぜ縦中横にするのかの理由の説明になり、またここに例の挙げられている以外の文字への一般化ができます。

ところで、順序を表す文字ではなく、同一の記号の場合で、縦書きで横倒しになってしまう場合もあり得ますね。この場合も正立させることになりますね。

> ピリオドなどの文字

「ピリオドなどの区切り文字を用いる場合、それらの区切り文字は縦中横にしない。区切り文字は順序を表す文字と本体を区切るのが目的であるからである。」などと理由を示して明確化。

また、ここのセクション、「場合がある」だらけで、どれが重要なのか、一般的なのか、プライオリティ付けがさっぱりわからない。という問題があるように思いました。

**Comment by taroyamamoto-451** [comment]
木田さん

リストについて、[questions/qa-upright-counters-in-vertical] Japanese vertical itemized lists #620に書いた内容とサンプルをアップデートしました。

参考まで

山本太郎
